routes/quantity.router.js

```diff
@@ -0,0 +1,7 @@
+const express = require("express");
+const router = express.Router();
+const { updateQuantityInCart } = require("../controllers/quantity.controller");
+
+router.route("").put(updateQuantityInCart);
```

Comment:
You need to add "/" for the default route, it's a common convention for router

Reply:
oops sorry didn't see this

Reply:
fixed it